packages/api/src/ai/services/memory.service.ts

```diff
@@ -20,9 +19,9 @@ export class MemoryService {
 
   getMemory(roomId: string, summary?: string): BufferMemory {
     if (!!summary) {
-      this.createMemoryWithSummary(roomId, summary);
+      void this.memoryMap.get(roomId).clear();
+      void this.createMemoryWithSummary(roomId, summary);
```

Comment:
these should be switched for `await`s since you don't care about the result, but you still want this code to run before the rest of the block